{"uid":"b3799910c2a783b9","name":"test_argocd_app_async_wait","fullName":"test_infra.TestInfra#test_argocd_app_async_wait","time":{"start":1725995006755,"stop":1725995245924,"duration":239169},"status":"failed","statusMessage":"Failed: Make cluster command failed with return code 2","statusTrace":"self = <test_infra.TestInfra object at 0x73f21e608e00>\ncmd = 'make argocd_app_async_wait'\n\n    def run_cmd(self, cmd):\n        try:\n>           result = subprocess.run([\"sh\", \"-c\", cmd], check=True, text=True, capture_output=True, cwd=\"../../\")\n\n/home/ci/mindwm-gitops/tests/mindwm_tests/test_infra.py:11: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\ninput = None, capture_output = True, timeout = None, check = True\npopenargs = (['sh', '-c', 'make argocd_app_async_wait'],)\nkwargs = {'cwd': '../../', 'stderr': -1, 'stdout': -1, 'text': True}\nprocess = <Popen: returncode: 2 args: ['sh', '-c', 'make argocd_app_async_wait']>\nstdout = \"make[1]: Entering directory '/home/ci/mindwm-gitops'\\n\\necho C2jVbApBGKNcct5f\\nC2jVbApBGKNcct5f\\ndocker run -i --rm -...a  Succeeded   Synced              namespace/redpanda unchanged\\nmake[1]: Leaving directory '/home/ci/mindwm-gitops'\\n\"\nstderr = 'Unable to use a TTY - input is not a terminal or the right kind of file\\n+ argocd login --plaintext --username admin ...websocket: close 1006 (abnormal closure): unexpected EOF\\nmake[1]: *** [Makefile:156: argocd_app_async_wait] Error 1\\n'\nretcode = 2\n\n    def run(*popenargs,\n            input=None, capture_output=False, timeout=None, check=False, **kwargs):\n        \"\"\"Run command with arguments and return a CompletedProcess instance.\n    \n        The returned instance will have attributes args, returncode, stdout and\n        stderr. By default, stdout and stderr are not captured, and those attributes\n        will be None. Pass stdout=PIPE and/or stderr=PIPE in order to capture them,\n        or pass capture_output=True to capture both.\n    \n        If check is True and the exit code was non-zero, it raises a\n        CalledProcessError. The CalledProcessError object will have the return code\n        in the returncode attribute, and output & stderr attributes if those streams\n        were captured.\n    \n        If timeout is given, and the process takes too long, a TimeoutExpired\n        exception will be raised.\n    \n        There is an optional argument \"input\", allowing you to\n        pass bytes or a string to the subprocess's stdin.  If you use this argument\n        you may not also use the Popen constructor's \"stdin\" argument, as\n        it will be used internally.\n    \n        By default, all communication is in bytes, and therefore any \"input\" should\n        be bytes, and the stdout and stderr will be bytes. If in text mode, any\n        \"input\" should be a string, and stdout and stderr will be strings decoded\n        according to locale encoding, or by \"encoding\" if set. Text mode is\n        triggered by setting any of text, encoding, errors or universal_newlines.\n    \n        The other arguments are the same as for the Popen constructor.\n        \"\"\"\n        if input is not None:\n            if kwargs.get('stdin') is not None:\n                raise ValueError('stdin and input arguments may not both be used.')\n            kwargs['stdin'] = PIPE\n    \n        if capture_output:\n            if kwargs.get('stdout') is not None or kwargs.get('stderr') is not None:\n                raise ValueError('stdout and stderr arguments may not be used '\n                                 'with capture_output.')\n            kwargs['stdout'] = PIPE\n            kwargs['stderr'] = PIPE\n    \n        with Popen(*popenargs, **kwargs) as process:\n            try:\n                stdout, stderr = process.communicate(input, timeout=timeout)\n            except TimeoutExpired as exc:\n                process.kill()\n                if _mswindows:\n                    # Windows accumulates the output in a single blocking\n                    # read() call run on child threads, with the timeout\n                    # being done in a join() on those threads.  communicate()\n                    # _after_ kill() is required to collect that and add it\n                    # to the exception.\n                    exc.stdout, exc.stderr = process.communicate()\n                else:\n                    # POSIX _communicate already populated the output so\n                    # far into the TimeoutExpired exception.\n                    process.wait()\n                raise\n            except:  # Including KeyboardInterrupt, communicate handled that.\n                process.kill()\n                # We don't call process.wait() as .__exit__ does that for us.\n                raise\n            retcode = process.poll()\n            if check and retcode:\n>               raise CalledProcessError(retcode, process.args,\n                                         output=stdout, stderr=stderr)\nE               subprocess.CalledProcessError: Command '['sh', '-c', 'make argocd_app_async_wait']' returned non-zero exit status 2.\n\n/usr/lib/python3.12/subprocess.py:571: CalledProcessError\n\nDuring handling of the above exception, another exception occurred:\n\nself = <test_infra.TestInfra object at 0x73f21e608e00>\n\n    @allure.story('Waiting for argocd mindwm application is ready')\n    @pytest.mark.dependency( name = 'argocd_app_async_wait', depends=['argocd_app_sync_async'], scope = \"session\")\n    def test_argocd_app_async_wait(self):\n>       self.run_cmd(\"make argocd_app_async_wait\")\n\n/home/ci/mindwm-gitops/tests/mindwm_tests/test_infra.py:40: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <test_infra.TestInfra object at 0x73f21e608e00>\ncmd = 'make argocd_app_async_wait'\n\n    def run_cmd(self, cmd):\n        try:\n            result = subprocess.run([\"sh\", \"-c\", cmd], check=True, text=True, capture_output=True, cwd=\"../../\")\n            print(\"Command Output:\", result.stdout)\n            assert result.returncode == 0, f\"Expected return code 0 but got {result.returncode}\"\n        except subprocess.CalledProcessError as e:\n            # If the command fails, capture the error and output\n            print(f\"Error executing '{cmd}': {e}\")\n            print(f\"Error Output: {e.stderr}\")\n>           pytest.fail(f\"Make cluster command failed with return code {e.returncode}\")\nE           Failed: Make cluster command failed with return code 2\n\n/home/ci/mindwm-gitops/tests/mindwm_tests/test_infra.py:18: Failed","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[],"afterStages":[],"labels":[{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"b3799910c2a783b9.json","parameterValues":[]}